bemani/backend/iidx/pendual.py

```diff
@@ -911,14 +911,13 @@ def handle_IIDX22pc_common_request(self, request: Node) -> Node:
         else:
             # If we aren't in an arcade, we turn off events
             timeshift_override = 0
+            event_phase = 0
             ccxbm_on = False
             omni_events = False
 
         if self.omnimix and (not omni_events):
             phase = 0
-        elif self.omnimix and omni_events or not self.omnimix:
```

Comment:
This looks like it was correct before. Taking this line out means that omni_events doesn't actually do anything.

Reply:
If omnievents is set to True, then the else clause executes. If omnimix is not True, then the else clause will always execute. I'm not sure I see the issue...

Reply:
Oh my god I'm stupid you're completely right.